### PR TITLE
fix: set upload limit to match FE limit

### PR DESCRIPTION
**Pull request by rebelchris**
After checking we actually set the FE limit to 5MB:
```
export const imageSizeLimitMB = 5;
```

So decided to set this one to match so we will catch it on both sides correctly.

### CODE REVIEW

File: src/graphorm/index.ts [hunk 1]
```diff
@@ -23,7 +24,6 @@ import { GQLBookmarkList } from '../schema/bookmarks';
 import { base64 } from '../common';
 import { GQLComment } from '../schema/comments';
 import { GQLUserPost } from '../schema/posts';
-import { UserNotification } from '../entity/notifications/UserNotification';
```
Comment (rebelchris) [inline src/graphorm/index.ts:26]: Just a cleaner import I noticed.
See above the `schema` ones, we should probably add a linter here for cleaner imports as we have on apps project.

File: src/index.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ export default async function app(
   // });
 
   app.register(MercuriusGQLUpload, {
-    maxFileSize: 1024 * 1024 * 2,
+    maxFileSize: 1024 * 1024 * 5,
```
Comment (idoshamun) [inline src/index.ts:132]: Let's do something bigger even

```suggestion
    maxFileSize: 1024 * 1024 * 20,
```

**Review by idoshamun — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (rebelchris) [reply src/index.ts:132]: I thought about that, but the FE is set to 5 anyway?
We can submit this regardless and see if we want to adjust it there?

Comment (idoshamun) [reply src/index.ts]: yes, probably good idea to change there as well :)

**Pull request merged**